.github/workflows/agw-workflow.yml

```diff
@@ -0,0 +1,311 @@
+---
+name: agw-workflow
+
+on:  # yamllint disable-line rule:truthy
+  push:
+    branches:
+      - master
+    paths:
+      - 'orc8r/**'
+      - 'lte/**'
+  pull_request:
+    branches:
+      - master
+jobs:
+  lte-test:
+    runs-on: ubuntu-latest
+    env:
+      MAGMA_ROOT: "${{ github.workspace }}"
+    steps:
+      - uses: actions/checkout@v2
+      - uses: actions/setup-python@v2
+        with:
+          python-version: '3.8.5'
+      - name: Install python3-aioeventlet from the magma apt repo
+        run: |
+            cd nms/app
+            # Install python3-aioeventlet from the magma apt repo
+            cat ${{ env.MAGMA_ROOT }}/orc8r/tools/ansible/roles/pkgrepo/files/jfrog.pub | sudo apt-key add -
+            echo "deb https://facebookconnectivity.jfrog.io/artifactory/list/dev-focal/ focal main" | sudo tee /etc/apt/sources.list.d/fbc.list
+            sudo apt-get update -y
+            sudo apt-get install -y python3-aioeventlet
+            sudo rm -rf /var/lib/apt/lists/*
+      - name: Install libraries and dependecies
+        run: |
+            mkdir -p /var/tmp/test_results
+            mkdir -p /var/tmp/codecovs
+            sudo apt-get update -y
+            sudo apt-get install -y libsystemd-dev pkg-config curl zip unzip
+            sudo apt-get install -y virtualenv python-babel python-dev build-essential python3-setuptools python-setuptools autogen autoconf libtool python3-apt python3-requests python3-pip python-protobuf
+            pip3 install setuptools==49.6.0
+      - name: Setup protoc3
+        run: |
+            sudo curl -Lfs https://github.com/google/protobuf/releases/download/v3.1.0/protoc-3.1.0-linux-x86_64.zip -o protoc3.zip
+            sudo unzip protoc3.zip -d protoc3
+            sudo mv protoc3/bin/protoc /bin/protoc
+            sudo chmod a+rx /bin/protoc
+            sudo mv protoc3/include/google /usr/include/
+            sudo chmod -R a+Xr /usr/include/google
+            sudo rm -rf protoc3.zip protoc3
+      - name: Setup swagger codegen
+        env:
+          CODEGEN_ROOT: "${{ env.MAGMA_ROOT }}/.codegen"
+          SWAGGER_CODEGEN_JAR: "${{ env.MAGMA_ROOT }}/.codegen/swagger-codegen-cli.jar"
+        run: |
+            sudo -E mkdir ${{ env.CODEGEN_ROOT }}
+            sudo -E wget https://repo1.maven.org/maven2/io/swagger/swagger-codegen-cli/2.2.3/swagger-codegen-cli-2.2.3.jar -O ${{ env.SWAGGER_CODEGEN_JAR }}
+            sudo -E chmod -R 755 ${{ env.CODEGEN_ROOT }}
+      - name: run test_all
+        env:
+          PYTHON_BUILD: "${{ env.MAGMA_ROOT }}/build"
+          PIP_CACHE_HOME: "${{ env.MAGMA_ROOT }}/.pipcache"
+          SWAGGER_CODEGEN_JAR: "${{ env.MAGMA_ROOT }}/.codegen/swagger-codegen-cli.jar"
+          MAGMA_DEV_MODE: 1
+          SKIP_SUDO_TESTS: 1
+        run: |
+            sudo -E make -C ${{ env.MAGMA_ROOT }}/lte/gateway/python test_all
+      - name: Publish Unit Test Results
+        uses: EnricoMi/publish-unit-test-result-action@v1
```

Comment:
You must upload the test artifacts, the other workflow will take care of publishing it

Reply:
are they in a different path? Can you show me an example?

Reply:
https://github.com/magma/magma/blob/8d1d33482689185ec38e63d5c1823ec77ea9f473/.github/workflows/cloud-workflow.yml#L152 with the path `/var/tmp/test_results`

Reply:
thank you!..